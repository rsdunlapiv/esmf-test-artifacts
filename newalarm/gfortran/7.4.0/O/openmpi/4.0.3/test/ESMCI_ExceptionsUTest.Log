2022-07-14 01:46:38
20220714 015636.279 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 015636.279 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 015636.279 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 015636.279 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 015636.279 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 015636.279 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 015636.279 INFO             PET0 Running with ESMF Version   : v8.3.0b13-21-g29f31e7579
20220714 015636.279 INFO             PET0 ESMF library build date/time: "Jul 14 2022" "01:44:35"
20220714 015636.279 INFO             PET0 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_7.4.0_openmpi_O_newalarm/esmf
20220714 015636.279 INFO             PET0 ESMF_COMM                   : openmpi
20220714 015636.279 INFO             PET0 ESMF_MOAB                   : enabled
20220714 015636.279 INFO             PET0 ESMF_LAPACK                 : enabled
20220714 015636.279 INFO             PET0 ESMF_NETCDF                 : enabled
20220714 015636.279 INFO             PET0 ESMF_PNETCDF                : disabled
20220714 015636.279 INFO             PET0 ESMF_PIO                    : enabled
20220714 015636.279 INFO             PET0 ESMF_YAMLCPP                : enabled
20220714 015636.279 INFO             PET0 Beginning Test, file ESMCI_ExceptionsUTest.C, line 63
20220714 015636.279 INFO             PET0 Beginning Test, file ESMCI_ExceptionsUTest.C, line 63NUMBER_OF_PROCESSORS 4
20220714 015636.279 INFO             PET0 PASS Bare throw Test, ESMCI_ExceptionsUTest.C, line 76
20220714 015636.279 INFO             PET0 PASS Deep throw Test, ESMCI_ExceptionsUTest.C, line 89
20220714 015636.279 INFO             PET0 PASS ClassNoDestructor throw Test, ESMCI_ExceptionsUTest.C, line 102
20220714 015636.279 INFO             PET0 PASS ClassWithDestructor throw Test, ESMCI_ExceptionsUTest.C, line 115
20220714 015636.279 INFO             PET0 Ending Test, file ESMCI_ExceptionsUTest.C, line 119
20220714 015636.279 INFO             PET0 Finalizing ESMF
20220714 015636.278 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 015636.278 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 015636.278 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 015636.278 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 015636.278 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 015636.278 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 015636.278 INFO             PET1 Running with ESMF Version   : v8.3.0b13-21-g29f31e7579
20220714 015636.278 INFO             PET1 ESMF library build date/time: "Jul 14 2022" "01:44:35"
20220714 015636.278 INFO             PET1 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_7.4.0_openmpi_O_newalarm/esmf
20220714 015636.278 INFO             PET1 ESMF_COMM                   : openmpi
20220714 015636.278 INFO             PET1 ESMF_MOAB                   : enabled
20220714 015636.278 INFO             PET1 ESMF_LAPACK                 : enabled
20220714 015636.278 INFO             PET1 ESMF_NETCDF                 : enabled
20220714 015636.278 INFO             PET1 ESMF_PNETCDF                : disabled
20220714 015636.278 INFO             PET1 ESMF_PIO                    : enabled
20220714 015636.278 INFO             PET1 ESMF_YAMLCPP                : enabled
20220714 015636.278 INFO             PET1 Beginning Test, file ESMCI_ExceptionsUTest.C, line 63
20220714 015636.278 INFO             PET1 Beginning Test, file ESMCI_ExceptionsUTest.C, line 63NUMBER_OF_PROCESSORS 4
20220714 015636.278 INFO             PET1 PASS Bare throw Test, ESMCI_ExceptionsUTest.C, line 76
20220714 015636.278 INFO             PET1 PASS Deep throw Test, ESMCI_ExceptionsUTest.C, line 89
20220714 015636.279 INFO             PET1 PASS ClassNoDestructor throw Test, ESMCI_ExceptionsUTest.C, line 102
20220714 015636.279 INFO             PET1 PASS ClassWithDestructor throw Test, ESMCI_ExceptionsUTest.C, line 115
20220714 015636.279 INFO             PET1 Ending Test, file ESMCI_ExceptionsUTest.C, line 119
20220714 015636.279 INFO             PET1 Finalizing ESMF
20220714 015636.279 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 015636.279 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 015636.279 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 015636.279 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 015636.279 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 015636.279 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 015636.279 INFO             PET2 Running with ESMF Version   : v8.3.0b13-21-g29f31e7579
20220714 015636.279 INFO             PET2 ESMF library build date/time: "Jul 14 2022" "01:44:35"
20220714 015636.279 INFO             PET2 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_7.4.0_openmpi_O_newalarm/esmf
20220714 015636.279 INFO             PET2 ESMF_COMM                   : openmpi
20220714 015636.279 INFO             PET2 ESMF_MOAB                   : enabled
20220714 015636.279 INFO             PET2 ESMF_LAPACK                 : enabled
20220714 015636.279 INFO             PET2 ESMF_NETCDF                 : enabled
20220714 015636.279 INFO             PET2 ESMF_PNETCDF                : disabled
20220714 015636.279 INFO             PET2 ESMF_PIO                    : enabled
20220714 015636.279 INFO             PET2 ESMF_YAMLCPP                : enabled
20220714 015636.279 INFO             PET2 Beginning Test, file ESMCI_ExceptionsUTest.C, line 63
20220714 015636.279 INFO             PET2 Beginning Test, file ESMCI_ExceptionsUTest.C, line 63NUMBER_OF_PROCESSORS 4
20220714 015636.279 INFO             PET2 PASS Bare throw Test, ESMCI_ExceptionsUTest.C, line 76
20220714 015636.279 INFO             PET2 PASS Deep throw Test, ESMCI_ExceptionsUTest.C, line 89
20220714 015636.279 INFO             PET2 PASS ClassNoDestructor throw Test, ESMCI_ExceptionsUTest.C, line 102
20220714 015636.279 INFO             PET2 PASS ClassWithDestructor throw Test, ESMCI_ExceptionsUTest.C, line 115
20220714 015636.279 INFO             PET2 Ending Test, file ESMCI_ExceptionsUTest.C, line 119
20220714 015636.279 INFO             PET2 Finalizing ESMF
20220714 015636.278 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 015636.278 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 015636.278 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 015636.278 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 015636.278 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 015636.278 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 015636.278 INFO             PET3 Running with ESMF Version   : v8.3.0b13-21-g29f31e7579
20220714 015636.278 INFO             PET3 ESMF library build date/time: "Jul 14 2022" "01:44:35"
20220714 015636.278 INFO             PET3 ESMF library build location : /glade/scratch/dunlap/esmf-test-dev/gfortran_7.4.0_openmpi_O_newalarm/esmf
20220714 015636.278 INFO             PET3 ESMF_COMM                   : openmpi
20220714 015636.278 INFO             PET3 ESMF_MOAB                   : enabled
20220714 015636.278 INFO             PET3 ESMF_LAPACK                 : enabled
20220714 015636.278 INFO             PET3 ESMF_NETCDF                 : enabled
20220714 015636.278 INFO             PET3 ESMF_PNETCDF                : disabled
20220714 015636.278 INFO             PET3 ESMF_PIO                    : enabled
20220714 015636.278 INFO             PET3 ESMF_YAMLCPP                : enabled
20220714 015636.278 INFO             PET3 Beginning Test, file ESMCI_ExceptionsUTest.C, line 63
20220714 015636.278 INFO             PET3 Beginning Test, file ESMCI_ExceptionsUTest.C, line 63NUMBER_OF_PROCESSORS 4
20220714 015636.278 INFO             PET3 PASS Bare throw Test, ESMCI_ExceptionsUTest.C, line 76
20220714 015636.278 INFO             PET3 PASS Deep throw Test, ESMCI_ExceptionsUTest.C, line 89
20220714 015636.278 INFO             PET3 PASS ClassNoDestructor throw Test, ESMCI_ExceptionsUTest.C, line 102
20220714 015636.278 INFO             PET3 PASS ClassWithDestructor throw Test, ESMCI_ExceptionsUTest.C, line 115
20220714 015636.278 INFO             PET3 Ending Test, file ESMCI_ExceptionsUTest.C, line 119
20220714 015636.278 INFO             PET3 Finalizing ESMF
