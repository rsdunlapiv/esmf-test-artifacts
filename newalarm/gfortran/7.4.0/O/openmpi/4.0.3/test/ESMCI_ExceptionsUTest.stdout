2022-07-14 01:46:38
Beginning Test, file ESMCI_ExceptionsUTest.C, line 63
Beginning Test, file ESMCI_ExceptionsUTest.C, line 63NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMCI_ExceptionsUTest.C, line 63
Beginning Test, file ESMCI_ExceptionsUTest.C, line 63NUMBER_OF_PROCESSORS 4
PASS Bare throw Test, ESMCI_ExceptionsUTest.C, line 76
PASS Deep throw Test, ESMCI_ExceptionsUTest.C, line 89
PASS ClassNoDestructor throw Test, ESMCI_ExceptionsUTest.C, line 102
PASS ClassWithDestructor throw Test, ESMCI_ExceptionsUTest.C, line 115
Ending Test, file ESMCI_ExceptionsUTest.C, line 119
PASS Bare throw Test, ESMCI_ExceptionsUTest.C, line 76
PASS Deep throw Test, ESMCI_ExceptionsUTest.C, line 89
PASS ClassNoDestructor throw Test, ESMCI_ExceptionsUTest.C, line 102
PASS ClassWithDestructor throw Test, ESMCI_ExceptionsUTest.C, line 115
Ending Test, file ESMCI_ExceptionsUTest.C, line 119
Beginning Test, file ESMCI_ExceptionsUTest.C, line 63
Beginning Test, file ESMCI_ExceptionsUTest.C, line 63NUMBER_OF_PROCESSORS 4
PASS Bare throw Test, ESMCI_ExceptionsUTest.C, line 76
PASS Deep throw Test, ESMCI_ExceptionsUTest.C, line 89
PASS ClassNoDestructor throw Test, ESMCI_ExceptionsUTest.C, line 102
PASS ClassWithDestructor throw Test, ESMCI_ExceptionsUTest.C, line 115
Ending Test, file ESMCI_ExceptionsUTest.C, line 119
Beginning Test, file ESMCI_ExceptionsUTest.C, line 63
Beginning Test, file ESMCI_ExceptionsUTest.C, line 63NUMBER_OF_PROCESSORS 4
PASS Bare throw Test, ESMCI_ExceptionsUTest.C, line 76
PASS Deep throw Test, ESMCI_ExceptionsUTest.C, line 89
PASS ClassNoDestructor throw Test, ESMCI_ExceptionsUTest.C, line 102
PASS ClassWithDestructor throw Test, ESMCI_ExceptionsUTest.C, line 115
Ending Test, file ESMCI_ExceptionsUTest.C, line 119
Ending Test, file ESMCI_ExceptionsUTest.C, line 119 PET 3 Test Elapsed Time 47.235 msec.
Ending Test, file ESMCI_ExceptionsUTest.C, line 119 PET 0 Test Elapsed Time 46.584 msec.
Ending Test, file ESMCI_ExceptionsUTest.C, line 119 PET 2 Test Elapsed Time 46.179 msec.
Ending Test, file ESMCI_ExceptionsUTest.C, line 119 PET 1 Test Elapsed Time 47.165 msec.
